worker.go

```diff
@@ -138,6 +139,7 @@ func (w *Worker) Work() {
 	for {
 		select {
 		case m := <-w.PacketChan:
+			fmt.Println("WTF ProcessMetric", m)
```

Comment:
Nope definitely not. Thanks! :) I thought I searched for all `'fmt` calls, but clearly I didn't check the diff carefully enough, thanks. :)